ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/CreateTableCommand.java

```diff
@@ -71,10 +72,10 @@ public DdlCommandResult run(final MutableMetaStore metaStore) {
         sourceName,
         schema,
         (keyColumnName.isEmpty())
-          ? null : SchemaUtil.getFieldByName(schema, keyColumnName).orElse(null),
+          ? Optional.empty() : SchemaUtil.getFieldByName(schema, keyColumnName),
         timestampExtractionPolicy,
         metaStore.getTopic(topicName),
-        stateStoreName, keySerde
+        stateStoreName, keySerdeFactory
```

Comment:
nit (not your code): two params on the same line.

Reply:
fixed.